Adyen/UI/Form/Items/Address/AddressViewModel.swift

```diff
@@ -30,7 +30,7 @@ public struct AddressViewModel {
     internal var labels: [AddressField: LocalizationKey]
     internal var placeholder: [AddressField: LocalizationKey]
     internal var optionalFields: [AddressField]
-    internal var scheme: [AddressFormScheme]
+    public var scheme: [AddressFormScheme]
```

Comment:
maybe annotate it with `@_spi(AdyenInternal)`, because merchants shouldn't access it, right? also maybe `private(set)`, we don't want outside code changing it, right?